pylint_django/transforms/transforms/django_db_models.py

```diff
@@ -1,7 +1,13 @@
+import random
+
 from django.core.exceptions import MultipleObjectsReturned \
     as MultipleObjectsReturnedException
 
 
+def __noop_get_or_create(self, *args, **kwargs):
+    return (Model(), bool(random.randint(0, 1)))
```

Comment:
Why use `random` ? Why not return just `True` or `False` ?

Reply:
This implementation is closer to the original one, which may return either `True` or `False` and that makes it more error-proof if in the future pylint gets smart enough to run checks based on exact return values of functions.

Reply:
Fair enough. I still need a reproducer/test though.